app/src/main/java/org/wikipedia/games/onthisday/OnThisDayGameNotificationManager.kt

```diff
@@ -111,11 +112,11 @@ class OnThisDayGameNotificationManager(private val activity: Activity) {
                     lang = null,
                     icon = null,
                     color = R.color.blue600,
-                    bodyIntent = OnThisDayGameActivity.newIntent(
-                        context = context,
-                        invokeSource = Constants.InvokeSource.NOTIFICATION,
-                        wikiSite = WikipediaApp.instance.wikiSite
-                    )
+                    bodyIntent = MainActivity.newIntent(context).apply {
+                        putExtra(Constants.INTENT_GO_TO_WIKI_GAME, true)
+                        flags = Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_NEW_TASK
+                        addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP)
+                    }
```

Comment:
Here's a case when you open the app from the notification, it will go back to the Explore feed when you re-open the app from the game screen, which is not expected.
https://github.com/user-attachments/assets/7bc2768a-ec69-4e07-be65-834ac846e447

If it is possible, can you please follow the way we handle the similar case in the `PageActivity`? 
```
if (app.haveMainActivity) {
    onBackPressed()
} else {
    pageFragment.goToMainActivity(tab = NavTab.EXPLORE, tabExtra = Constants.INTENT_EXTRA_GO_TO_MAIN_TAB)
}
```